waffle-doppelganger/README.md

```diff
@@ -0,0 +1,113 @@
+![Ethereum Waffle](https://raw.githubusercontent.com/EthWorks/Waffle/master/docs/source/logo.png)
+
+[![CircleCI](https://circleci.com/gh/EthWorks/Waffle.svg?style=svg)](https://circleci.com/gh/EthWorks/Waffle)
+[![](https://img.shields.io/npm/v/@ethereum-waffle/doppelganger.svg)](https://www.npmjs.com/package/@ethereum-waffle/doppelganger)
+
+_doppelgänger /ˈdɒp(ə)lˌɡaŋə,ˈdɒp(ə)lˌɡɛŋə/ - an apparition or double of a living person_
+
+# @ethereum-waffle/doppelganger
+
+Library for mocking smart contract dependencies during unit testing.
+
+## Installation
+In the current version of waffle (v2.x.x) you will install this package as a dependency of the main waffle package - `ethereum-waffle`.
+
+```
+yarn add --dev ethereum-waffle
+npm install --save-dev ethereum-waffle
+```
+
+If you want to use this package directly please install it via:
+```
+yarn add --dev @ethereum-waffle/doppelganger
+npm install --save-dev @ethereum-waffle/doppelganger
+```
+
+## Usage
+
+Create a instance of fake contract providing the ABI/interface of the smart contract you want to mock:
+
+```js
+import {doppelganger} from '@ethereum-waffle/doppelganger';
+
+...
+
+const mockContract = await doppelganger(wallet, contractAbi);
```

Comment:
lets rename `doppelganger` -> `mockContract`
@sz-piotr Agreed?

Reply:
But then you have this stupid situation where you cannot write
```
const mockContract = await mockContract(wallet, abi)
```

I think it should be named with a verb like `deployMockContract` to go in line with `deployContract`

Reply:
👍